main.css

```diff
@@ -784,6 +817,429 @@ body {
 }
 
 .type__social-icon {
+    width: 10px;
+    height: 10px;
+}
+
+.type-hidden__driver {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    display: none;
+}
+
+.type-hidden__text {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+}
+
+.business__partnerimage-source-delta {
+    display: none;
+}
+
+.business__partnerimage-source-ascension {
+    display: none;
+}
+
+@media only screen and (min-width: 380px) {
+.let-ride__buttons {
+    flex-direction: row;
+}
+}
+
+@media only screen and (min-width: 768px) {
+
+.ride__container {
```

Comment:
formatting issues, should have tab before

Reply:
+